handlers/get_teacher.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	runtime "github.com/ahab94/ApnaSchool"
+	"github.com/ahab94/ApnaSchool/gen/restapi/operations"
+	"github.com/go-openapi/runtime/middleware"
+)
+
+// NewGetTeacher handles a request for retrieving teacher
+func NewGetTeacher(rt *runtime.Runtime) operations.GetTeacherHandler {
+	return &getTeacher{rt: rt}
+}
+
+type getTeacher struct {
+	rt *runtime.Runtime
+}
+
+// Handle the get teacher request
+func (d *getTeacher) Handle(params operations.GetTeacherParams) middleware.Responder {
+	_, err := d.rt.Service().RetrieveTeacher(params.ID)
+	if err != nil {
+		return operations.NewGetTeacherNotFound()
+	}
+	return operations.NewGetTeacherOK()
```

Comment:
We need to send the teacher back in the response.

Reply:
Well, im sorry but i dont know how to that.

Reply:
i did try. that was my first approach. but i couldnt figure it out